keyboards/alu84/rules.mk

```diff
@@ -53,4 +23,4 @@ SLEEP_LED_ENABLE ?= yes  # Breathing sleep LED during USB suspend
 NKRO_ENABLE ?= yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 BACKLIGHT_ENABLE ?= yes  # Enable keyboard backlight functionality
 AUDIO_ENABLE ?= no
-RGBLIGHT_ENABLE ?= yes
\ No newline at end of file
+RGBLIGHT_ENABLE ?= yes
```

Comment:
get rid of ?= maybe?

Reply:
This whole section is using `?=`. That should be changed IMO.

Reply:
I didn't want to mess with the build options sections here, so I left that stuff alone, as much as it bothers me 😄